### ISSUE TITLE
Full installation not working

### ISSUE THREAD

**Issue by arty-hlr** [ardagnir/athame#68]
Hi,

Just wanted to post that the full installation of athame as my default readline didn't work for me. I gathered this was because of a readline problem, but I couldn't manage to solve it. Here is the output of `ldd $(which bash)`:

```
linux-vdso.so.1 (0x00007fffe9d90000)
libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007f926bf8c000)
libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f926bf87000)
libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f926bdc6000)
/lib64/ld-linux-x86-64.so.2 (0x00007f926c114000)
```

And indeed I don't see libreadline. The `./readline_athame_setup.sh --use_sudo` does work, but then running `./bash_readline_setup.sh --use_sudo` gives me this error:

```
(...)
config.status: creating po/Makefile
config.status: executing default commands
make: *** empty string invalid as file name.  Stop.

Make failed: Are you sure you have readline 8 installed? readline_athame_setup.sh installs readline 8 patched with athame. You may want to run it first.
This may also fail if you have a versionless libreadline.so symlinked to libreadline.so.7
```

I tried installing myself readline 8, with the same output. However, the local install worked, and my workaround was to change my shell to the local one, but still, I'm curious as to what happened there!

If you have any hints or solutions for this, I'd be grateful to try it :)

Thanks in advance,

Florian

**Comment by cevhyruz** [comment]
same here!

**Comment by ardagnir** [comment]
This should be fixed in v1.2 (and master branch).
